### PR TITLE
cling: secure URLs

### PR DESCRIPTION
**Pull request by vszakats**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by fxcoudert** [comment]
```
00:08:03 ==> brew readall --aliases homebrew/core
00:08:13 ==> FAILED
00:08:13 Error: Invalid formula: /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/cling.rb
00:08:13 Version value must be a string; got a NilClass ()
```

File: Formula/cling.rb [hunk 1]
```diff
@@ -16,15 +16,15 @@ class Cling < Formula
   depends_on "cmake" => :build
 
   resource "clang" do
-    url "http://root.cern.ch/git/clang.git",
-        :tag => "cling-patches-r302975",
-        :revision => "1f8b137c7eb06ed8e321649ef7e3f3e7a96f361c"
+    url "https://github.com/vgvassilev/clang.git",
```
Comment (leonklingele) [inline Formula/cling.rb]: Why should we trust you here?

**Comment by vszakats** [comment]
Anyway, this is too messy. There is this insecure cern.ch repo that holds patched versions of outdated LLVM & clang. Then there are the new GitHub repos under a lead maintainer with different patches. I could not find canonical documentation about how to build the project, so let's just stick with what we already have here. The core repo upgrade seems to be genuine and safe to keep though. [ Ideally a documented revision of mainline LLVM & clang would be best to vendor, along with one cling-specific patch for each, from a genuine source. ]

File: Formula/cling.rb (hunk 1 above)
Comment (leonklingele) [reply Formula/cling.rb]: Oops, misread your GitHub username as **vgvassilev**. I'm so sorry.

Comment (vszakats) [reply Formula/cling.rb]: Oh, OK — no worries!

**Pull request merged**